db/config.json

```diff
@@ -1,7 +1,12 @@
 {
   "development": {
     "dialect": "postgres",
-    "use_env_variable": "DATABASE_URL"
+    "use_env_variable": "DATABASE_URL",
```

Comment:
for some reason, I was getting some local connection issues with sequelize and this seems to fix that up.  I think there's code somewhere that's running and never resolving properly?  We are going to need to update sequelize soon.